dags/common/task_group_with_timeout/task_group_with_timeout.py

```
@@ -63,47 +63,56 @@ def __init__(
       self,
       group_id,
       timeout: timedelta,
-      is_teardown: bool = False,
+      as_teardown_of: BaseOperator | None = None,
       **kwargs,
```

Ruff (0.16.1)

[warning] Missing type annotation for `**kwargs`

(ANN003)